liquidity-module/index.md

```diff
@@ -359,10 +420,14 @@ Make sure to replace `cosmosxxxxx` with your address.
 You can also withdraw tokens from the pool you have created.
 
 ```bash
-gaiad tx liquidity withdraw 1 100pool-id --from username --chain-id cosmoshub-testnet --gas-prices "0.025stake" --node https://rpc.testnet.cosmos.network:443
+gaiad tx liquidity withdraw 1 100pool-id --from username --chain-id cosmoshub-testnet --gas-prices "0.025uphoton" --node https://rpc.testnet.cosmos.network:443
 ```
 
-- Replace `pool-id` with your pool id that you can see on [https://api.testnet.cosmos.network/cosmos/bank/v1beta1/balances/cosmosxxx](https://api.testnet.cosmos.network/cosmos/bank/v1beta1/balances/cosmosxxx)
+Make sure to replace the first `1` with your `pool id` obtained in previous step.
+
+- Replace `pool-id` with the id that you can see on [https://api.testnet.cosmos.network/cosmos/bank/v1beta1/balances/cosmosxxx](https://api.testnet.cosmos.network/cosmos/bank/v1beta1/balances/cosmosxxx)
```

Comment:
oh, I need to learn more here, we have `100pool-id` so we leave the `100` and just replace the words `pool-id` with a value?

Reply:
```suggestion
- Replace `pool-id` with the pool id that you can see on [https://api.testnet.cosmos.network/cosmos/bank/v1beta1/balances/cosmosxxx](https://api.testnet.cosmos.network/cosmos/bank/v1beta1/balances/cosmosxxx)
```

Reply:
I purposely used `id` instead of `pool-id`. Else people will get confused.